doc/man3/EVP_PKEY_set1_RSA.pod

```diff
@@ -62,13 +62,16 @@ see L<openssl_user_macros(7)>:
 EVP_PKEY_get_base_id() returns the type of I<pkey>. For example
 an RSA key will return B<EVP_PKEY_RSA>.
 
-EVP_PKEY_get_id() returns the actual OID associated with I<pkey>.
-Historically keys using the same algorithm could use different OIDs.
-For example an RSA key could use the OIDs corresponding to
+EVP_PKEY_get_id() returns the actual NID associated with I<pkey>
+if I<pkey> is implemented outside of a L<provider(7)>.
```

Comment:
`also outside of`

Reply:
what about `only if the I<pkey> type isn't implemented just in a L<provider(7)>.`

Reply:
so done in the latest commit